frb_example/with_flutter/rust/src/bridge_generated.web.rs

```diff
@@ -0,0 +1,327 @@
+#![allow(
+    non_camel_case_types,
+    unused,
+    clippy::redundant_closure,
+    clippy::useless_conversion,
+    clippy::unit_arg,
+    clippy::double_parens,
+    non_snake_case
+)]
+// AUTO GENERATED FILE, DO NOT EDIT.
+// Generated by `flutter_rust_bridge`.
+
+use crate::api::*;
+use flutter_rust_bridge::*;
+
+// Section: imports
+
+// Section: wire functions
+
+#[wasm_bindgen]
+pub fn wire_draw_mandelbrot(
+    port_: MessagePort,
+    image_size: Box<[JsValue]>,
+    zoom_point: Box<[JsValue]>,
+    scale: f64,
+    num_threads: i32,
+) {
+    FLUTTER_RUST_BRIDGE_HANDLER.wrap(
+        WrapInfo {
+            debug_name: "draw_mandelbrot",
+            port: Some(port_),
+            mode: FfiCallMode::Normal,
+        },
+        move || {
+            let api_image_size = image_size.wire2api();
+            let api_zoom_point = zoom_point.wire2api();
+            let api_scale = scale.wire2api();
+            let api_num_threads = num_threads.wire2api();
+            move |task_callback| {
+                draw_mandelbrot(api_image_size, api_zoom_point, api_scale, api_num_threads)
+            }
+        },
+    )
+}
```

Comment:
Again, just like the Dart counterpart - seems that there are some similar things between .rs and .web.rs, is it possible to extract something?